shared/src/main/scala/pl/metastack/metatime/Offset.scala

```diff
@@ -15,6 +17,34 @@ object Formatter {
   def branch(ranges: Seq[(Range, Any)], fallback: Any): Formatter = ???
 }
 
-trait Offset[T] {
-  def format: String = ???
+trait Offset[+T] {
```

Comment:
Actually I tried that but I got this error:

`\Offset.scala:47: type arguments [Component] do not conform to trait Offset's type parameter bounds [T <: pl.metastack.metatime.Component]
[error]   def apply[Component](value: Component): Offset[Component] = new Offset[Component] {
[error]   def apply[Component](value: Component): Offset[Component] = new Offset[Component] {`

that's why I didn't include T <: Compoent, any idea how to solve it further?